ct-app/test/test_config.yaml

```
@@ -2,50 +2,52 @@
+    retrieve_peers: 30
+    retrieve_channels: 30
+    retrieve_balances: 30
+
+    open_channels: Off
+    fund_channels: Off
+    close_old_channels: Off
+    close_pending_channels: Off
+    close_incoming_channels: Off
+
+    get_total_channel_funds: Off
     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)